src/ID3Frames.js

```diff
@@ -545,3 +545,30 @@ module.exports.COMR = {
         return tag
     }
 }
+
+module.exports.GEOB = {
+    create: (data) => {
+        if (!(data instanceof Array)) {
+            data = [data]
+        }
+
+        return Buffer.concat(data.map((geob) => {
+            return new ID3FrameBuilder("GEOB")
+                .appendNullTerminatedValue(geob.mimeType ? geob.mimeType : '', 0x00)
+                .appendNullTerminatedValue(geob.filename ? geob.filename : '', 0x00)
+                .appendNullTerminatedValue(geob.description ? geob.description : '', 0x00)
+                .appendStaticValue(geob.data)
+                .getBuffer()
+        }))
+    },
+
+    read: (buffer) => {
+        const reader = new ID3FrameReader(buffer)
+        return {
+            mimeType: reader.consumeNullTerminatedValue('string', 0x00),
+            filename: reader.consumeNullTerminatedValue('string', 0x00),
+            description: reader.consumeNullTerminatedValue('string', 0x00),
```

Comment:
`filename` and `description` are encoded according to the encoding byte,

```suggestion
            mimeType: reader.consumeNullTerminatedValue('string', 0x00),
            filename: reader.consumeNullTerminatedValue('string'),
            description: reader.consumeNullTerminatedValue('string'),
```